readme.txt

```diff
@@ -8,23 +8,23 @@ Stable tag: trunk
 Enable Two-Factor Authentication using time-based one-time passwords (OTP, Google Authenticator), Universal 2nd Factor (FIDO U2F, YubiKey), email and backup verification codes.
 
 == Description ==
-
-Use the "Two-Factor Options" section under "Users" → "Your Profile" to enable and configure one or multiple two-factor authentication providers for your account:
-
-- Email codes
-- Time Based One-Time Passwords (TOTP)
-- FIDO Universal 2nd Factor (U2F)
-- Backup Codes
-- Dummy Method (only for testing purposes)
+
+Use the "Two-Factor Options" section under "Users" → "Your Profile" to enable and configure one or multiple two-factor authentication providers for your account:
+
+- Email codes
```

Comment:
Looks like there are some whitespace changes in here. Could you please remove them? Or was this intentional?

Reply:
I didn't touched any line. 
Simply nano readme.txt and added the 3rd screenshot text and then save. I'm not sure why this is happening

Reply:
OK, I'll check that when I do the GitHub repo URL update in the next pull request.